test/sass/typography.spec.scss

```diff
@@ -159,3 +159,99 @@
   }
 
 }
+
+@include describe('@mixin label-as-content') {
+
+  @include it('generates correct default styles') {
+
+    @include assert() {
+
+      @include output {
+        @include typography.label-as-content();
+      }
+
+      @include expect {
+        color: rgb(102, 102, 102);
+        letter-spacing: 0.5px;
+        font-family: "Noto Sans", Arial, Helvetica, sans-serif;
+        font-size: 0.6875rem;
+        font-weight: 400;
+        line-height: 2.1818181818;
+        text-transform: uppercase;
+      }
+
+    }
+
+  }
+
+  @include it('allows the color to be overridden') {
+
+    @include assert() {
+
+      $_acolor: rgb(2, 119, 189);
+
+      @include output {
+        @include typography.label-as-content($color: $_acolor);
+      }
+
+      @include expect {
+        color: $_acolor;
+        letter-spacing: 0.5px;
+        font-family: "Noto Sans", Arial, Helvetica, sans-serif;
+        font-size: 0.6875rem;
+        font-weight: 400;
+        line-height: 2.1818181818;
+        text-transform: uppercase;
+      }
+
+    }
+
+  }
+
+  @include it('allows the text-transform to be overridden') {
+
+    @include assert() {
+
+      $_acolor: rgb(102, 102, 102);
```

Comment:
Don't need this here. Line 222 should just specify the colour directly. As discussed, the colour only needs to be in a variable if it's used in both the `output` and the `expect` blocks. Here, it's only in the `expect` block so there's no duplication to remove, so there's no need a variable. (For an appropriate implementation using a variable see [within the assert block on line 236](https://github.com/libero/storybook/pull/31/files#diff-34e711f734d561d250826e618e148f3eR236).)